### DISCUSSION TITLE
Paragraphs missing from struct when using Smart Format (Go Deepgram SDK)

### DISCUSSION THREAD

**Discussion by RobinLbt** [deepgram/community#578] [General help] [answered]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

When using the official Deepgram Go SDK (version v1.1.1) with Smart Format & Nova 2, the response doesn't contain the paragraphs

```go
	prerecordedTranscriptionOptions := interfaces.PreRecordedTranscriptionOptions{
		Model:          "nova-2-video",
		DetectLanguage: true,
		Diarize:        true,
		SmartFormat:    true,
	}

res, err := deepgram.DeepgramPrerecordedClient.FromURL(ctx, url, prerecordedTranscriptionOptions)

logger.Info("Deepgram transcription result (paragraphs)", "result", res.Results.Channels[0].Alternatives[0].Paragraphs)
```

When I try the same API call directly from Postman (with the same query params, API key, and file URL), the resulting JSON contains paragraphs

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?model=nova-2-video&diarize=true&detect_language=true&smart_format=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

c372e51b-d867-4ec9-b369-1f12143cfc4c

### If possible, please attach your code or paste it into the text box.

	prerecordedTranscriptionOptions := interfaces.PreRecordedTranscriptionOptions{
		Model:          "nova-2-video",
		DetectLanguage: true,
		Diarize:        true,
		SmartFormat:    true,
	}

res, err := deepgram.DeepgramPrerecordedClient.FromURL(ctx, url, prerecordedTranscriptionOptions)

logger.Info("Deepgram transcription result (paragraphs)", "result", res.Results.Channels[0].Alternatives[0].Paragraphs)

### If possible, please attach an example audio file to reproduce the issue.

https://github.com/deepgram/community/assets/25883677/baf1a4bb-f794-4b85-b3f5-51ba0179ca50

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
Hi @RobinLbt 

Did you enable the feature? https://github.com/deepgram/deepgram-go-sdk/blob/main/pkg/client/interfaces/types-prerecorded.go#L37

**Comment by davidvonthenen** [comment]
Ooooof. Looks like there is a bug on the return type. Investigating!

**Comment by davidvonthenen** [comment, marked as answer]
Yep, there is! Should be fixed in this PR:
https://github.com/deepgram/deepgram-go-sdk/pull/174

There is a release pending today for another fix.

**Reply by RobinLbt** [reply]
Wow, that was fast!
Thank you @dvonthenen

**Reply by davidvonthenen** [reply]
This release should have the fix:
https://github.com/deepgram/deepgram-go-sdk/releases/tag/v1.1.2
